python/cudf/cudf/core/column/decimal.py

```diff
@@ -83,8 +83,9 @@ def binary_operator(self, op, other, reflect=False):
                 scale=scale, precision=Decimal64Dtype.MAX_PRECISION
             )  # precision will be ignored, libcudf has no notion of precision
             result = libcudf.binaryop.binaryop(self, other, op, output_type)
-            result.dtype.precision = _binop_precision(
-                self.dtype, other.dtype, op
+            result.dtype.precision = min(
```

Comment:
Should the `min` go inside `_binop_precision`, or is the unbounded `_binop_precision` result useful somehow?

Reply:
Good question, I was thinking either this should go inside `_binop_precision` or a note should be added to `_binop_precision`. I opted for the latter because part of me was thinking this can be useful if we ever want to interface with other systems that has a different precision limit (T-Sql for example), this can come in handy. 

On the other hand, if this function is set to use elsewhere inside cudf codebase, it's probably a better idea to encode the clipping within.

Reply:
I'd go ahead and put the `min` inside `_binop_precision`, because we only have one use case for it right now. we can always come back and modify it later if we need to.